doc/en/example/markers.rst

```diff
@@ -71,6 +73,28 @@ Or the inverse, running all tests except the webtest ones:
 
     ===================== 3 passed, 1 deselected in 0.12s ======================
 
+.. _`marker_keyword_expression_example`:
+
+Additionally, you can restrict a test run to only run tests matching one or multiple marker
+keyword arguments, e.g. to run only tests marked with ``device`` and the specific ``serial="123"``:
+
+.. code-block:: pytest
+
+    $ pytest -v -m 'device(serial="123")'
+    =========================== test session starts ============================
+    platform linux -- Python 3.x.y, pytest-8.x.y, pluggy-1.x.y -- $PYTHON_PREFIX/bin/python
+    cachedir: .pytest_cache
+    rootdir: /home/sweet/project
+    collecting ... collected 4 items / 3 deselected / 1 selected
+
+    test_server.py::test_something_quick PASSED                          [100%]
+
+    ===================== 1 passed, 3 deselected in 0.12s ======================
+
+.. note:: Only keyword argument matching is supported in marker expressions.
+
+.. note:: Only `int`, (unescaped) `str`, `bool` & `None` values are supported in marker expressions.
```

Comment:
Single backticks are for Markdown. In RST they often won't behave as you think. Either use double backticks or declare explicit roles:
```suggestion
.. note:: Only :class:`int`, (unescaped) :class:`str`, :class:`bool` & :data:`None` values are supported in marker expressions.
```

Reply:
Yep, using markdown always, ...changed to double back ticks everywhere.

Reply:
It's usually better to use explicit roles, though.